Emby.Server.Implementations/Playlists/PlaylistManager.cs

```diff
@@ -566,19 +569,28 @@ public async Task RemovePlaylistsAsync(Guid userId)
         public async Task UpdatePlaylist(PlaylistUpdateRequest request)
         {
             var playlist = GetPlaylistForUser(request.Id, request.UserId);
+            if (playlist is null)
+            {
+                throw new ArgumentException($"No playlist exists with Id {request.Id}");
+            }
 
             if (request.Ids is not null)
             {
+                var user = _userManager.GetUserById(request.UserId);
+                if (user is null)
+                {
+                    throw new ArgumentException($"No user exists with Id {request.UserId}");
+                }
+
                 playlist.LinkedChildren = [];
                 await UpdatePlaylistInternal(playlist).ConfigureAwait(false);
 
-                var user = _userManager.GetUserById(request.UserId);
                 await AddToPlaylistInternal(request.Id, request.Ids, user, new DtoOptions(false)
                 {
                     EnableImages = true
                 }).ConfigureAwait(false);
 
-                playlist = GetPlaylistForUser(request.Id, request.UserId);
+                playlist = GetPlaylistForUser(request.Id, request.UserId)!;
```

Comment:
Avoid this because this is not properly handling the null case

Reply:
It has already been checked by the `GetPlaylistForUser` call above, so it is safe to assume that UserId is valid. But I understand if you want to make it more explicit, I can handle the null value there too.

Reply:
We're getting a playlist we've just updated, so it is safe so assume it will not be null. But I can make it more explicit and handle it again if it is more compliant to standards.